src/bson_value.ts

```diff
@@ -10,8 +11,16 @@ export abstract class BSONValue {
     return BSON_MAJOR_VERSION;
   }
 
+  [Symbol.for('nodejs.util.inspect.custom')](
```

Comment:
Let's mark this as internal

Reply:
… even though it is just implementing a standardized API of the platform? What are you trying to hide from users here?

Reply:
Good point here. Please disregard.